nasa/earthdata-search
# EDSC-3361: Adding deprecated parameter modal

# Overview

### What is the feature?

Adds the ability to display a modal when a deprecated parameter is used.

### What is the Solution?

Display the modal

# Testing

### Reproduction steps

- **Environment for testing:** ANY
- **Collection to test with:** N/A

1. Using the redux dev tools, trigger the following action:
 `{ type: 'TOGGLE_DEPRECATED_PARAMETER_MODAL', payload: true }`
2. Confirm the modal opens
3. Run the next action:
 `{ type: 'TOGGLE_DEPRECATED_PARAMETER_MODAL', payload: false }`
4. Confirm the modal is closed
### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist
